### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Mr-Zippers** [Cyfrin/foundry-full-course-cu#2897] [General]
I've been trying to make my first transaction using the faucet recommended but every single time I try it I get a message that says native tokens require the user to hold at least one link on Ethereum Mainnet. What am I doing wrong please?

**Comment by EngrPips** [comment]
You are not doing anything wrong, Getting sepolia eth has been difficult lately. You can try this faucet https://sepolia-faucet.pk910.de/

**Reply by chainflash** [reply]
You can follow the two sites I wrote below :
https://cloud.google.com/application/web3/faucet/ethereum/sepolia
https://faucets.chain.link/sepolia

**Reply by EngrPips** [reply]
Great heads-up.

**Reply by chainflash** [reply]
Thanks I appreciate it

**Reply by Mr-Zippers** [reply]
Ok, thanks. Let me try the new faucet

**Reply by EngrPips** [reply]
Alright, Lets know how that plays out.

**Reply by Mr-Zippers** [reply]
Thanks guys, it worked. I just made my first transaction and I'm super excited.

**Reply by EngrPips** [reply]
Awesome, keep crushing it.

**Reply by Vladimir-Silenko** [reply]
tryin' to get tockens in jan. 2025 returns me message: native tokens require the user to hold at least one link on Ethereum Mainnet.

**Reply by EngrPips** [reply]
Try this https://sepolia-faucet.pk910.de/ or this https://cloud.google.com/application/web3/faucet/ethereum/sepolia @Vladimir-Silenko

**Reply by Xlroo** [reply]
> Try this https://sepolia-faucet.pk910.de/ or this https://cloud.google.com/application/web3/faucet/ethereum/sepolia @Vladimir-Silenko

Thank you so much. I just started learning about web3 and these faucets sent me into a rabbit hole.

**Reply by EngrPips** [reply]
Anytime, Friend. It will get easier with time.

**Reply by Mengbuxiu** [reply]
> Try this https://sepolia-faucet.pk910.de/ or this https://cloud.google.com/application/web3/faucet/ethereum/sepolia @Vladimir-Silenko

Thank you so much for your help. I got 1 score in the first link, the second one is what i neeed! THX!

**Reply by EngrPips** [reply]
Awesome.

**Reply by adennie** [reply]
The google one worked for me today, thanks. The other one requires a Gitcoin Passport, which I haven't set up.

**Reply by EngrPips** [reply]
It is fine, I am glad you have something that works at least.

**Reply by Dikewonsi** [reply]
Thanks @Mengbuxiu. You're awesome.

**Comment by Mengbuxiu** [comment]
You reply so fast, are you online 24/7?
---- Replied Message ----
From ***@***.***> Date 02/21/2025 03:59 To ***@***.***> Cc ***@***.***>***@***.***> Subject Re: [Cyfrin/foundry-full-course-cu] Help (Discussion #2897) 
It is fine, I am glad you have something that works at least.

**Reply by EngrPips** [reply]
Not really, I just always try to not leave anyone hanging as far as I can still respond.
